### PR TITLE
completed api-registration

### PR DESCRIPTION
**Pull request by Quintin-Russell**
image

### CODE REVIEW

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    returning "userId", "username", "createdAt"
+  `;
+  argon2.hash(password)
+    .then(res => {
+      const params = [username, res];
+      return db.query(sql, params);
+    })
+    .then(result => {
+      const response = result.rows;
+      res.status(201).json(response);
```
Comment (scbowler) [inline api-registration/server/index.js:41]: Here you are sending back an array with a single object, we only want to send back the object without the array. So you would update line 40:

```JAVASCRIPT
const response = result.rows[0];
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed. Let me know if you have any questions.

File: api-registration/server/index.js (hunk 1 above)
Comment (Quintin-Russell) [reply api-registration/server/index.js:41]: done

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**